src/math/Spherical.js

```diff
@@ -1,12 +1,13 @@
+
+import * as MathUtils from './MathUtils.js';
+
 /**
- * Ref: https://en.wikipedia.org/wiki/Spherical_coordinate_system
+ * Ref: https://en.wikipedia.org/wiki/Spherical_coordinate_system (mathematics convention)
  *
- * The polar angle (phi) is measured from the positive y-axis. The positive y-axis is up.
- * The azimuthal angle (theta) is measured from the positive z-axis.
+ * phi (the polar angle) is measured from the positive y-axis. The positive y-axis is up.
+ * theta (the azimuthal angle) is measured from the positive z-axis.
  */
 
-import * as MathUtils from './MathUtils.js';
```

Comment:
Moving the import statement at the top is fine but I don't see how the textual changes are an improvement, tbh.

Do you mind reverting these changes?

Reply:
I can revert the changes, however let me try to convince you that they are a good thing before :)

I updated the texts so that the first subject of the docstring matches the constructor's argument name (phi and theta).

I specified _mathematics convention_ because the linked wikipedia page shows the physics convention as the first image and it may be confusing to see inverted theta and phi.

That said let me know if I should revert the changes, I have no strong opinions about that.

Reply:
I've made some small changes and the merge the PR.